### PR TITLE
Nightwatch changes for Toolkit 2.0

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Update for the newest changes in [terra-toolkit v2.0 release](https://github.com/cerner/terra-toolkit/pull/8).

Additionally, changes were made for [terra-base v2.0 release](https://github.com/cerner/terra-core/pull/738).

@cerner/terra

### CODE REVIEW

File: generators/app/templates/terra-clinical-package.json [hunk 1]
```diff
@@ -24,12 +24,12 @@
   "peerDependencies": {
```
Comment (bjankord) [inline generators/app/templates/terra-clinical-package.json]: Can you add the following before peerDependencies:

```js
"devDependencies": {
  "terra-props-table": "^1.4.0"
},
```

File: generators/app/templates/terra-consumer-package.json [hunk 2]
```diff
@@ -25,12 +25,12 @@
   "peerDependencies": {
```
Comment (bjankord) [inline generators/app/templates/terra-consumer-package.json]: Can you add the following before peerDependencies:

```js
"devDependencies": {
  "terra-props-table": "^1.4.0"
},
```

File: generators/app/templates/terra-package.json [hunk 3]
```diff
@@ -23,12 +23,12 @@
   "peerDependencies": {
```
Comment (bjankord) [inline generators/app/templates/terra-package.json]: Can you add the following before peerDependencies:

```js
"devDependencies": {
  "terra-props-table": "^1.4.0"
},
```

**Comment by emilyrohrbough** [comment]
Resolves https://github.com/cerner/generator-terra-module/issues/68

**Pull request merged**